release-tools/is-lp-fips-build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -x
+
+# check for triggers that indicate a build on LP of the snapd-fips snap
+# recipe https://launchpad.net/~fips-cc-stig/fips-cc-stig/+snap/snapd-fips
+#
+# - git origin https://git.launchpad.net/~snappy-dev/snapd/+git/snapd
+# - build path: /build/snapd-fips
+
+if ! git remote get-url origin | grep "git.launchpad.net" >&2 ; then
+    echo "false"
+    exit 0
+fi
+
+if ! echo "$PWD" | grep '^/build/snapd-fips/' >&2 ; then
```

Comment:
Where is this coming from? What makes `build/snapd-fips`?

Reply:
it's from the snap recipe in LP: https://launchpad.net/~fips-cc-stig/fips-cc-stig/+snap/snapd-fips, LP checks out the code at `/build/snapd-fips/`

Reply:
Perhaps leave a comment